src/applications/static-pages/social-share-links.js

```diff
@@ -5,8 +5,12 @@ Open share links in a new modal window
 export function openShareLink(shareLinks) {
   const hasNavigatorShare = navigator.share !== undefined;
   const metaTitle = document.querySelector('title').content;
-  const metaDescription = document.querySelector("meta[name='description']")
-    .content;
+  const metaDescriptionElement = document.querySelector(
+    "meta[name='description']",
+  );
+  const metaDescription = metaDescriptionElement
+    ? metaDescriptionElement.content
+    : '';
```

Comment:
Could use the `??` operator -
```js
const metaDescription = metaDescriptionElement?.content ?? '';
```

Reply:
Had to read up about that operator! Cool!! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
We are getting _fancy_ 🕺 💃